### DISCUSSION TITLE
How to get error from FirebaseAuthentication If an error occurs ?

### DISCUSSION THREAD

**Discussion by george-martinec** [capawesome-team/capacitor-firebase#234] [Q&A] [answered]
Hi,

Is there a possibility to get an error message, for example from android if an authentication error occurs ?

For example the account is deactivated in firebase dashboard and we can see in android Logcat:

```
E/FirebaseAuthentication: signInWithCredential failed.
    com.google.firebase.auth.FirebaseAuthInvalidUserException: The user account has been disabled by an administrator.
```

I am using a function: `await FirebaseAuthentication.signInWithGoogle()`

**Comment by robingenz** [comment, marked as answer]
Unfortunately not at the moment. There is already an issue for this: https://github.com/capawesome-team/capacitor-firebase/issues/25
